### ISSUE TITLE
Localization is needed

### ISSUE THREAD

**Issue by lewadedun** [alketola/harbour-ostos#5]
Could you support www.transifex.com, please?

**Comment by alketola** [comment]
Good idea to study further, and see how far we can get with a zero budget.

**Comment by lewadedun** [comment]
https://www.transifex.com/pricing/ - it's free to Open Source projects that have no funding, revenue, and/or commercialization model
